common/Dockerfile.tests

```diff
@@ -0,0 +1,19 @@
+FROM python:3.6.1-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+COPY common/requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY data_models/dist/data-refinery-models-* data_models/
+
+# Get the latest version from the dist directory.
+RUN pip install data_models/$(ls data_models -1 | tail -1)
```

Comment:
You're definitely right about this. I fixed this once but I forgot to port it everywhere: https://github.com/data-refinery/data_refinery/blob/master/foreman/Dockerfile#L14